test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -149,6 +149,8 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
 				// This is because the entire config map is stored in `last-applied` annotation for tracking.
 				// The workaround is to use server side apply by passing `--server-side` flag to kubectl apply.
 				// More on server side apply here: https://kubernetes.io/docs/reference/using-api/server-side-apply/
+				// TODO(CAPI 1.8?): Need a PR to re-add argument support to this type.
+				// It was removed in https://github.com/kubernetes-sigs/cluster-api/commit/b4349fecaa626865e71b058a8b01e0377fb9e444
 				Args: []string{"--server-side"},
```

Comment:
We'll probably want to open a PR upstream to support passing server-side apply to kubectl now that the `Args` field is gone and `CreateOrUpdateOpts` doesn't currently cover all available arguments.